Meta/Lagom/Tools/CodeGenerators/JSSpecCompiler/Parser/SpecParser.cpp

```diff
@@ -173,4 +176,31 @@ ParseErrorOr<void> SpecFunction::parse_definition(XML::Node const* element)
     return {};
 }
 
+SpecParsingStep::SpecParsingStep()
+    : CompilationStep("parser"sv)
+{
+}
+
+SpecParsingStep::~SpecParsingStep() = default;
+
+void SpecParsingStep::run(TranslationUnitRef translation_unit)
+{
+    auto filename = translation_unit->filename;
+
+    auto file = Core::File::open_file_or_standard_stream(filename, Core::File::OpenMode::Read).release_value_but_fixme_should_propagate_errors();
+    m_input = file->read_until_eof().release_value_but_fixme_should_propagate_errors();
```

Comment:
same question here about this dangling reference.

Future refactor might try to use MappedFile instead.